src/menu_put.c

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   menu_put.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/01/06 12:35:59 by gwolf             #+#    #+#             */
+/*   Updated: 2024/01/06 14:29:58 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "menu_put.h"
+
+void	ft_menu_put_text(t_render *render)
+{
+	t_putinfo	put;
+
+	put.mlx_ptrs = render->mlx_ptrs;
+	put.pos = (t_vec2i){20, 20};
+	put.col = render->menu.font_col;
+	if (render->show_menu == false)
+		return (ft_put_str(put,
+				"Press I to show menu"));
+	mlx_put_image_to_window(render->mlx_ptrs.mlx_ptr,
+		render->mlx_ptrs.win_ptr, render->mlx_ptrs.veil.ptr, 0, 0);
+	put.pos = ft_put_mode(put, render->mode);
+	if (render->menu.cur_page % 3 == 0)
```

Comment:
Do we have MACROS for the page numbers? So instead of comparing to a number we have a description which is more readable

Reply:
Yes there is an enum for the pages.  The thing is that mod the third page would make less sense.
The pages are 0 scene, 1 light, 2 cam, 3 scene control, 4, light control, 5 cam control.
With mod3 I can differentiate between scene, light or cam (control or not).
I could make an extra macro NUM_OF_MODES or something to mask the three?